docker-compose.yaml

```diff
@@ -0,0 +1,24 @@
+version: "3"
+services:
+  llama-cpp:
+    build:
+      context: .
+    hostname: llama-cpp-server
+    command: "python3 -m llama_cpp.server --model /models/mistral-7b-instruct-v0.2.Q4_K_M.gguf --host 0.0.0.0 --port 8090"
```

Comment:
Do you think we could factor out ports etc. into `.env` file? [0]
It would simplify maintenance of the entire setup, although it's obviously an optional thing.

[0]https://docs.docker.com/compose/compose-file/12-interpolation/

Reply:
sure, I can do that